bcrs-ui/babel.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  presets: [
+    '@vue/cli-plugin-babel/preset'
+  ]
+}
```

Comment:
We need to find out why this is different than coops-ui version. Does this one work? Does the coops-ui one need to be updated?
I think that if this requires more than 1 hour of research then we should create a new ticket.

Reply:
Good question!  I will look into it.

Reply:
At first glance, the vue/cli-plugin-babel/preset is a wrapper for a few things like Babel 7 + babel-loader + @vue/babel-preset-app. I think this just saves the time of adding these manually as options in the babel configuration. It does indeed work, but i don't know enough about babel and webpack to say for sure that the Coops-ui should be updated.

Reply:
I had a quick look at `@vue/babel-preset-app/index.js` and it appears to set a bunch of default app config -- including some that appear different from what we do in coops-ui.

I think we should "resolve this conversation" but keep an eye on what gets built (ie, npm run build) and how it's loading and running in different browsers, etc. If things seem incorrect then we can revisit this config.